python/ray/experimental/state/api.py

```diff
@@ -567,6 +652,26 @@ def get_objects(
     timeout: int = DEFAULT_RPC_TIMEOUT,
     _explain: bool = False,
 ) -> List[ObjectState]:
+    """Get objects by id.
+
+    There could be more than 1 entry returned since an object could be
+    referenced at different places.
+
+    Args:
+        id: Id of the object
+        address: The IP address and port of the head node. Defaults to
+            `http://localhost:8265` if not set.
+        timeout: Max timeout value for the http/gRPC requests made.
+        _explain: Print the API information such as API latency or
+            failed query information.
+
+    Returns:
+        List of :ref:`ObjectState <state-api-schema-obj>`.
```

Comment:
Is it still a list although only 1 entry is returned btw?

Reply:
It is still a list.